src/pages/gd/Portfolio/index.tsx

```diff
@@ -447,7 +447,18 @@ const Portfolio = () => {
                     </Table>
                 </Card>
             )}
-            {network !== 'production' && <SavingsAccount account={account} />}
+            {network !== 'production' &&
+                (
+                    Object.keys(SupportedV2Networks).filter((v) =>
+                        isNaN(Number(v))
+                    ) as (keyof typeof SupportedV2Networks)[]
+                ).map((key) => {
+                    return (
+                        <Fragment key={key}>
+                            <SavingsAccount requiredChain={SupportedV2Networks[key]} account={account} />
+                        </Fragment>
+                    )
+                })}
```

Comment:
```suggestion
                ).map((key) => (
                        <Fragment key={key}>
                            <SavingsAccount requiredChain={SupportedV2Networks[key]} account={account} />
                        </Fragment>
                    ))}
```

Reply:
no need return here
arrow function returns value of the expression if it's one-liner

const a = () => 124 // a() returns 124
const b = async () => 123 // await b() returns 123

Reply:
ur right, changed